federatedscope/autotune/fts/utils.py

```diff
@@ -0,0 +1,523 @@
+from datetime import datetime
+import GPy
```

Comment:
The introduced dependency should be mentioned in the "requirement" declaration (see /environment/).